dashboard/src/shared/utils.ts

```diff
@@ -85,6 +87,10 @@ export function getPluginIcon(plugin?: Plugin | string) {
     switch (plugin?.name) {
       case PluginNames.PACKAGES_HELM:
         return helmIcon;
+      case PluginNames.PACKAGES_FLUX:
+        return fluxIcon;
+      case PluginNames.PACKAGES_KAPP:
+        return carvelIcon;
```

Comment:
Created #3748

Reply:
Well, right now we aren't displaying anything, but, yep I agree we should. 
Out of curiosity (and vocabulary in the advertising world), for a background thing like this, I prefer using "isologos" (like helm, text is not separable) and "isotypes" (just the icon, not text) rather than the whole "imagotype" (icon+text, like flux or the OLM).

Some alternatives:

- A1: Use the tag for those non-helm pkgs
![image](https://user-images.githubusercontent.com/11535726/141282439-c9ec9bc8-6ebb-43d3-a181-448611b8f871.png)

- A2: add a tooltip
![image](https://user-images.githubusercontent.com/11535726/141283518-0786af59-2749-4743-8924-5d1342224b75.png)

- A3: add a tag, same style with the plugin name
![image](https://user-images.githubusercontent.com/11535726/141284054-29d854b6-231c-49a2-a4c4-b53d03d255c9.png)

- A4:  add a tag, different style with the plugin name (simplified)
![image](https://user-images.githubusercontent.com/11535726/141284330-57b859ce-e248-49a9-81d9-72a11ef76a8e.png)

WDYT, @absoludity ?  I'd go with A4, not sure about the color yet (bc we should also see how it looks like in dark mode)

Reply:
Yep, I agree with the simplified additional tag, probably capitalized? (`Helm` or `Flux` or `Carvel`) of A4. Not sure about capitalization given it's not normally used for tags, but in this case it's for a brand/name.

RE color/style: It may be enough if it's always in the same position (bottom right)? (but don't mind either way).